doc/stdlib/dates.rst

```diff
@@ -136,6 +136,11 @@ alternatively, you could call ``using Dates`` to bring all exported functions in
    Returns a DateTime corresponding to the user's system
    time as UTC/GMT.
 
+.. function:: eps(::DateTime) -> Millisecond
+              eps(::Date) -> Day
+
+   Returns ``Millisecond(1)`` for ``DateTime`` values and ``Day(1)`` for ``Date`` values.
+
```

Comment:
Since this is to do with the old help system I'm not really the best person to ask (not sure who is, come to think of it). The best I can suggest is to wait, since anything like this should be ironed out as we transition to the new system.